documentation/operators/src/nodes/proxy-configuration.md

```diff
@@ -374,6 +378,261 @@ where you should read out
 ```
 or just point your browser to the URI which you set above, such as https://nym-exit.<YOUR_DOMAIN>
 
+
+
+
+```
+
+## Web Secure Socket Setup
+
+- INTRO WHY
+- VAR LEGEND
+
+
+Before you start, don't forget to register a DNS and configure a record for your VPS with `nym-node`.
+
+
+If you haven't configure reversed proxy before, start with [*Preliminary steps* chapter](#preliminary-steps) below and only then move to WSS setup. If you have reversed proxy already running, you can skip *Premliminary steps* and begin to setup WSS directly. Remember that there may be some unique variables and customisation depending on the way your reversed proxy is done which you may have to adjust when installing WSS in order to make it work.
+
+We documented two options for node operators to setup WSS for `nym-node`:
+
+1. [Using a script](#using-a-script)
+2. [Step by step](#step-by-step)
+
+**Variables Explanation**
+
+This guide contains several variables. Substitute them with your own value, without `<>` brackets. Here is a list of variables we used below.
+
+| Variable              | Description                                                    | Syntax example                                            |
```

Comment:
Nice this is a cool approach. Maybe we can do something like this as a standalone page for the operators docs as well, a 'variable glossary'